SharesCalculation.py

```diff
@@ -0,0 +1,17 @@
+import ApplicationOutput as op
+import SpouseAndParentSharesCalculation as sps
+def calculate(amount, sons, daughters, mother, father, wife):
+    if amount > 0:
```

Comment:
ambiguous function name without docstrings.

Reply:
I'll take care about it..